npm/svelte/src/mount.ts

```diff
@@ -56,19 +48,24 @@ const getComponentDisplayName = <T extends SvelteComponent>(Component: SvelteCon
  *
  * @see {@link https://on.cypress.io/mounting-svelte} for more details.
  */
-export function mount<T extends SvelteComponent> (
-  Component: SvelteConstructor<T>,
-  options: MountOptions<T> = {},
-): Cypress.Chainable<MountReturn<T>> {
+export function mount (
+  Component: Component<Record<string, any>, Record<string, any>, any>,
+  options: Omit<MountOptions, 'target'> & {log?: boolean} = {},
+): Cypress.Chainable<MountReturn> {
+  // In Svelte 5, the component name is no longer easily discoverable and logs as "wrapper"
+  // so we default the logging of it to false as it doesn't provide a lot of value
+  options.log = options.log || false
```

Comment:
depending on how Svelte is bundled has an impact here, where the `name` of the component can only be determined to be `wrapper`.  For instance, the `name` in open mode is `wrapper`, but in `run` mode it's the full component name. This like has to do with the `development` version of svelte being used in `open` mode but `production` being used in `run`. 

Because of this, I have default the mount log off by default as when developing tests it doesn't provide a lot of value to just say `mount<wrapper>`